// sapper/internal/manifest-server.mjs
// This file is generated by Sapper — do not edit it!
import * as component_0 from "../../../routes/index.svelte";
import * as component_1 from "../../../routes/peliculas/[id]/index.svelte";
import * as root_comp from "../../../routes/_layout.svelte";
import error from "../../../routes/_error.svelte";

const d = decodeURIComponent;

export const manifest = {
	server_routes: [
		
	],

	pages: [
		{
			// index.svelte
			pattern: /^\/$/,
			parts: [
				{ name: "index", file: "index.svelte", component: component_0 }
			]
		},

		{
			// peliculas/[id]/index.svelte
			pattern: /^\/peliculas\/([^/]+?)\/?$/,
			parts: [
				null,
				{ name: "peliculas_$id", file: "peliculas/[id]/index.svelte", component: component_1, params: match => ({ id: d(match[1]) }) }
			]
		}
	],

	root_comp,
	error
};

// Allow miltiple sapper instances, dirs relative to sapper build instaed of cwd
// export const build_dir = ${JSON.stringify(build_dir)};
export const build_dir = require("path").resolve(__dirname, "../");

// export const src_dir = ${JSON.stringify(src_dir)};
export const src_dir = require("path").resolve(__dirname, "../../../src");

export const dev = true;
